charts/argo-rollouts/templates/argo-rollouts-clusterrole.yaml

```diff
@@ -2,11 +2,10 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: {{ .Release.Name }}-clusterrole
+  name: {{ include "argo-rollouts.fullname" . }}
```

Comment:
Can we use `{{ include "argo-rollouts.fullname" . }}-clusterrole` like https://github.com/argoproj/argo-helm/pull/752/files#diff-0f556b9a4c151c044e3a45c0ae4cdc6037ec46e49f940a25051f0a3e1c6b50a1R59

Reply:
For me it does not make sense to name a ClusterRole object `foobar-clusterrole`. Also it doesn't make sense to name a Secret a `foo-secret`. WDYT?

Reply:
Agreed. Maybe the change should be made to the argo-rollouts/repo.

Reply:
Shall we open an issue over there or do I revert this change back?

Reply:
@mkilchhofer , done: https://github.com/argoproj/argo-rollouts/issues/1231